lib/internal/promisify.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
```

Comment:
Fixed, thanks. Although I think such cases should ideally be fixed in a one-off way to fulfill with an object, not an array.